api/tsoa/swagger.json

```diff
@@ -6,18 +6,42 @@
 		"requestBodies": {},
 		"responses": {},
 		"schemas": {
+			"StatusResponse": {
+				"properties": {
+					"message": {
+						"type": "string"
+					}
+				},
+				"required": [
+					"message"
+				],
+				"type": "object",
+				"additionalProperties": false
+			},
 			"User": {
 				"properties": {
+					"pass": {
+						"type": "string"
+					},
+					"salt": {
+						"type": "string"
+					},
```

Comment:
Again, this implies that the API gives us the password hash and current active JWT token for users when a User is returned.